lib/membrane_flac_plugin/membrane_flac_parser.ex

```diff
@@ -33,7 +33,16 @@ defmodule Membrane.FLAC.Parser do
 
   @impl true
   def handle_init(_ctx, opts) do
-    {[], opts |> Map.from_struct() |> Map.merge(%{parser: nil, input_pts: nil})}
+    {[],
+     opts
+     |> Map.from_struct()
+     |> Map.merge(%{
+       parser: nil,
+       input_pts: nil,
+       current_pts: nil,
+       meta_queue: [],
+       frame_duration: nil
+     })}
```

Comment:
This will be more readable
```suggestion
    state =
     opts
     |> Map.from_struct()
     |> Map.merge(%{
       parser: nil,
       input_pts: nil,
       current_pts: nil,
       meta_queue: [],
       frame_duration: nil
     }
   
   {[], state}
```